apps/retrospective/services/task.go

```diff
@@ -66,6 +66,50 @@ func (service TaskService) Get(id string, retroID string, sprintID string) (task
 	return &tasks[0], http.StatusOK, nil
 }
 
+// MarkDone ...
+func (service TaskService) MarkDone(id string, retroID string, sprintID string) (task *retroSerializers.Task, status int, err error) {
+	db := service.DB
+	var sprint retroModels.Sprint
+	err = db.Model(&retroModels.Sprint{}).
+		Where("id = ?", sprintID).
+		Scan(&sprint).Error
+
```

Comment:
what if sprint doesn't exist, shouldn't 404 error be ideal then?

Reply:
Done